school_transport/__manifest__.py

```diff
@@ -0,0 +1,28 @@
+# See LICENSE file for full copyright and licensing details.
+
+{
+    "name": "Transport Management",
+    "version": "14.0.1.0.0",
+    "author": "Serpent Consulting Services Pvt. Ltd.",
+    "website": "http://www.serpentcs.com",
+    "license": "AGPL-3",
+    "category": "School Management",
+    "complexity": "easy",
+    "summary": "A Module For Transport & Vehicle Management In School",
+    "depends": ["school"],
```

Comment:
check for the FLEET compatability. It isa nearfit now.
@AnuKPatel @ForamSerpentCS